bindings/experimental/distrdf/python/DistRDF/Backends/Dask/Backend.py

```diff
@@ -12,16 +12,19 @@
 from __future__ import annotations
 
 import os
-from typing import Any, Dict, Optional, TYPE_CHECKING
+from typing import Any, Dict, List, Optional, Callable, TYPE_CHECKING, Union, Tuple
+import math
+import ROOT
 
 from DistRDF import DataFrame
 from DistRDF import HeadNode
+from DistRDF import Ranges
```

Comment:
Do you need the `Ranges` module to create some object or is it just for type hints? In the latter case, protect the import with an `if TYPE_CHECKING`